blog/2022-summer-of-code.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Google Summer of Code 2022
+date: 2022-03-07T00:00:00.000Z
+authors:
+  - name: erickzhao
+    url: 'https://github.com/erickzhao'
+    image_url: 'https://github.com/erickzhao.png?size=96'
+  - name: sofianguy
+    url: 'https://github.com/sofianguy'
+    image_url: 'https://github.com/sofianguy.png?size=96'
+  - name: dsanders11
+    url: 'https://github.com/dsanders11'
+    image_url: 'https://github.com/dsanders11.png?size=96'
+  - name: vertedinde
+    url: 'https://github.com/vertedinde'
+    image_url: 'https://github.com/vertedinde.png?size=96'
+slug: 2022-summer-of-code
+---
+
+The Electron team is excited to announce that we will be participating in Google Summer of Code for the first time this year!
+
+---
+
+## What is Google Summer of Code?
+
+Google Summer of Code (GSoC) is a yearly mentoring program connecting open source software projects with potential contributors. Previously open only to students, anyone ages 18 and up can now register for GSoC.
+
+For more information, check out the [Summer of Code homepage](https://summerofcode.withgoogle.com/).
+
+## How do I sign up?
+
+Are you interested in collaborating with Electron? If you are a new or beginner open source contributor, we welcome you to apply!
+
+In order to be selected as an Electron contributor for Google Summer of Code, you will need to submit an application. Applications will open on **April 4th, 2022** and close on **April 19th, 2022**. [You can follow updates for Google: Summer of Code application guidelines here](https://opensource.googleblog.com/2021/11/expanding-google-summer-of-code-in-2022.html).
+
+Want to apply? First, check out the [five project idea drafts](https://docs.google.com/document/d/1hJBiju0185Tga99WF3023u0uHgnfCG-J1DfzmhYHca0/edit?usp=sharing) that we have prepared. All of the listed ideas are currently open for proposals. We are also open to accepting new ideas that are not on the proposed project list.
+
+Your application should include:
+* Your proposal, which is a written document that describes in detail what you plan to achieve over the course of the summer.
+* A copy of your resume/cover letter.
+* Any additional technical samples or information you’d like to share.
```

Comment:
```suggestion
* Your background as a developer. If you have a resume, please include a copy, otherwise tell us about your past experience with an emphasis on relevant technical experience.
```